.github/workflows/pixi_auto_update.yml

```
@@ -20,12 +20,15 @@ jobs:
       - name: Update pixi lock file
         run: |
           rm pixi.lock
-          pixi install
+          pixi global install pixi-diff-to-markdown
+          printf "Update pixi dependencies to the latest version\n" >> diff.md
+          pixi update --no-install --json | pixi exec pixi-diff-to-markdown >> diff.md
+
       - uses: peter-evans/create-pull-request@v6
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           branch: update/pixi-lock
           title: Update pixi lock file
           commit-message: "Update `pixi.lock`"
-          body: Update pixi dependencies to the latest version.
+          body-path: diff.md
```

```suggestion
          body-path: diff.md
          add-paths: pixi.lock
```

Taken from the example, sounds like a good addition to me: https://github.com/peter-evans/create-pull-request#action-inputs

---

thanks, updated.